changelogs/CHANGELOG-1.4.md

```diff
@@ -1,3 +1,22 @@
+## v1.4.1
+### 2020-05-26
```

Comment:
This needs to be updated.

Reply:
🤦 Thanks! I had done this a couple times and had it right in the first attempt, but not this one.